website/source/docs/providers/aws/r/app_cookie_stickiness_policy.html.markdown

```diff
@@ -15,20 +15,20 @@ Provides an application cookie stickiness policy, which allows an ELB to wed its
 ```
 resource "aws_elb" "lb" {
     name = "test-lb"
-	  availability_zones = ["us-east-1a"]
-	  listener {
-	      instance_port = 8000
-		    instance_protocol = "http"
-		    lb_port = 80
-		    lb_protocol = "http"
-	  }
+	availability_zones = ["us-east-1a"]
+	listener {
+	  instance_port = 8000
+	  instance_protocol = "http"
+	  lb_port = 80
+	  lb_protocol = "http"
+    }
 }
 
 resource "aws_app_cookie_stickiness_policy" "foo" {
-	  name = "foo_policy"
-	  load_balancer = "${aws_elb.lb.id}"
-	  lb_port = 80
-	  cookie_name = "MyAppCookie"
+	name = "foo_policy"
```

Comment:
Hm, you're right, AWS docs are not always matching with the reality. I will open a support case, encourage them to update it. I've been already reporting some RDS validation, which was out of date the other day.

Until they document it, I think we should just validate against what the error says (i.e. alphanumeric + dash). Feel free to test certain edge cases, like two dashes, dashes at either end of the name, but I'd guess these will be allowed, based on the error message.